### ISSUE TITLE
Implement Greek ligatures

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [georgd/EB-Garamond#174]
It would awesome to see Greek ligatures available in your font.

See: https://en.wikipedia.org/wiki/Greek_ligatures

**Comment by georgd** [comment]
This is more or less a duplicate of https://github.com/georgd/EB-Garamond/issues/151. As I said there, the whole set of the Grecs du Roi ligatures is out of scope for me. I'll be busy enough for many² months with merging the octaviopardo branch, adapting the EBG08 for the optical size axis and creating the four 48pt masters for the opposite extreme. 

If anybody volunteers I'll be happy to accept pull requests.

**Comment by KrasnayaPloshchad** [comment]
So I think this should be developed as separate branch, and this could be made in parallel with EB Garamond font.
